build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
+  echo "Checking if html-build is up to date..."
+  ORIGIN_URL=$(git remote get-url origin)
+  git fetch -q $ORIGIN_URL master
```

Comment:
I'm not sure I know enough to be concerned about changing FETCH_HEAD :). What does that do?

If this is indeed meant to not mess with origin, then it seems OK to be hiding it behind verbose and silencing it by default. It doesn't output any more useful information, like number of commits in the update, or a progress bar?